models/productDao.js

```diff
@@ -0,0 +1,24 @@
+const { appDataSource } = require("./dataSource");
+
+const getdetailProduct = async (productId) => {
+  try {
+    return await appDataSource.query(
+      `
+	SELECT p.name, 
+         p.price, 
+         p.description
+	FROM products as p
+	WHERE p.id = ?
+	`,
+      [productId]
+    );
+  } catch (err) {
+    const error = new Error("INVALID_DETAILDATA");
+    error.statusCode = 500;
```

Comment:
상태코드 500은 에러처리를 안 했다는 의미가 될 수 있으므로 400으로 수정해 주세요.
```suggestion
    error.statusCode = 400;
```